array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,9 @@
+/* exported includes */
+function includes(array, count) {
```

Comment:
To avoid confusion within your source code, please make sure your parameter names match those from the spec:

---

![image](https://user-images.githubusercontent.com/16143771/231523428-99bc28dc-0698-4ee5-a782-f06f69793f6a.png)